bitnami/discourse/templates/secrets-discourse.yaml

```diff
@@ -11,17 +12,11 @@ metadata:
   {{- end }}
 type: Opaque
 data:
+  {{- if not .Values.discourse.existingSecret }}
```

Comment:
This is not needed, we only create this secret if `.Values.discourse.existingSecret` is not provided.